pom.xml

```diff
@@ -124,6 +124,10 @@
             <artifactId>json-schema-validator</artifactId>
             <version>2.2.14</version>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-reactive-jsonb</artifactId>
```

Comment:
We already depend on `quarkus-resteasy-reactive-jackson` and I know from experience that mixing both is not a good idea.